bootstrap.sh

```diff
@@ -105,13 +105,44 @@ install_protoc() {
   case $(get_arch) in
       aarch64)  local target=aarch_64;;
       x86_64)  local target=x86_64;;
+      arm64) case "$platform" in
+          osx) local use_homebrew=1;;
+          *) echo "ERROR: unsupported architecture"; exit 1;;
+      esac;;
       *)   echo "ERROR: unsupported architecture"; exit 1;;
   esac
 
-  # This is how we'd download directly from source:
-  # wget https://github.com/protocolbuffers/protobuf/releases/download/v$version/protoc-$version-$platform-${target}.zip
-  $VTROOT/tools/wget-retry "${VITESS_RESOURCES_DOWNLOAD_URL}/protoc-$version-$platform-${target}.zip"
-  unzip "protoc-$version-$platform-${target}.zip"
+  # TODO (ajm188): remove this branch after protoc includes signed protoc binaries for M1 Macs.
+  if [[ "$use_homebrew" -eq 1 ]]; then
+    cat >&2 <<WARNING
+WARN: Protobuf does not have a protoc binary for arm64 macos.
+Checking for homebrew installation. Altertatively, you may install the x86-64
+version if you have Rosetta installed; your mileage may vary.
+
+See https://github.com/protocolbuffers/protobuf/issues/9397.
+WARNING
+
+    if [[ -z "$(command -v brew)" ]]; then
+      echo "Could not find \`brew\` command. Please install homebrew and retry." >&2;
+      exit 1;
+    fi
+
+    brew install protobuf;
```

Comment:
For linux we always install a well-known version (`$version`). Should we do the same for OSX and pin a known version?

Reply:
you actually ... can't 🙈 

with homebrew you just get latest, which is one of the bigger downsides of going this way vs the "just download the x86 and ask them to use rosetta" that i originally advocated

e.g.

```
➜  dev brew list protobuf | head -n 1
/opt/homebrew/Cellar/protobuf/3.19.4/INSTALL_RECEIPT.json
➜  dev brew search protobuf
==> Formulae
protobuf ✔          protobuf-c          protobuf@3.6        swift-protobuf
➜  dev brew uninstall protobuf >/dev/null
➜  dev brew install protobuf@3.19.4
Warning: No available formula with the name "protobuf@3.19.4". Did you mean protobuf@3.6?
==> Searching for similarly named formulae...
This similarly named formula was found:
protobuf@3.6
To install it, run:
  brew install protobuf@3.6
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.
➜  dev
```